Build failure on Mac OS X 10.11.6 with Xcode 8.0

There are 2 problems to build on Mac OS X 10.11.6 with Xcode 8.0 and wxWindows 3.0.2 (also 3.1.0).

## 1. Linking error with libpng

```
Apple Mach-O Linker (ld) Error Group
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: library not found for -lpng
```

This can be fixed by adding `-L/usr/local/lib` to "pwsafe - Build Settings - Linking - Other Linker Flags" to use brewed libpng, because Mac OS X 10.11 may not bundle libpng itsself.

## 2. Another linking error

```
Apple Mach-O Linker (ld) Error Group
Undefined symbols for architecture x86_64:
  "CPasswordSubset::CPasswordSubset(wxWindow*, std::__1::basic_string<wchar_t, std::__1::char_traits<wchar_t>, S_Alloc::SecureAlloc<wchar_t> > const&, int, wxString const&, wxPoint const&, wxSize const&, long)", referenced from:
      PasswordSafeFrame::DoPasswordSubset(CItemData&) in mainEdit.o
  "CPasswordSubset::~CPasswordSubset()", referenced from:
      PasswordSafeFrame::DoPasswordSubset(CItemData&) in mainEdit.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I don't know how to resolve this problem.

--- a/Xcode/generate-configs
+++ b/Xcode/generate-configs
@@ -32,7 +32,7 @@ my $libflags;
 foreach(split /\s+/, `$wxconfig --libs --unicode=yes --debug=$dbgflag`) {
 	$libflags .= $_." " if /\A-[lL]|\.a\Z/;
 }
-print "OTHER_LDFLAGS = ", $libflags, "\n";
+print "OTHER_LDFLAGS = -L/usr/local/lib ", $libflags, "\n";
 
 my $cxxflags;
 foreach(split /\s+/, `$wxconfig --cxxflags --unicode=yes --debug=$dbgflag`) {

--- a/Xcode/pwsafe-xcode6.xcodeproj/project.pbxproj
+++ b/Xcode/pwsafe-xcode6.xcodeproj/project.pbxproj
@@ -21,6 +21,7 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		6FF5D4A11DA14EE80032F5B6 /* passwordsubset.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 6FF5D49D1DA14AB20032F5B6 /* passwordsubset.cpp */; };
 		A2D181461C8FF86C0018AE03 /* media.cpp in Sources */ = {isa = PBXBuildFile; fileRef = A2D181451C8FF86C0018AE03 /* media.cpp */; };
 		A2FE25801C5ACEFD00210C36 /* AES.cpp in Sources */ = {isa = PBXBuildFile; fileRef = A2FE257E1C5ACEFD00210C36 /* AES.cpp */; };
 		A2FE258D1C5ACF7500210C36 /* Item.cpp in Sources */ = {isa = PBXBuildFile; fileRef = A2FE25811C5ACF7500210C36 /* Item.cpp */; };
@@ -177,6 +178,8 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		6FF5D49D1DA14AB20032F5B6 /* passwordsubset.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = passwordsubset.cpp; sourceTree = "<group>"; };
+		6FF5D49E1DA14AB20032F5B6 /* passwordsubset.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = passwordsubset.h; sourceTree = "<group>"; };
 		A2D181451C8FF86C0018AE03 /* media.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = media.cpp; sourceTree = "<group>"; };
 		A2FE257E1C5ACEFD00210C36 /* AES.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = AES.cpp; sourceTree = "<group>"; };
 		A2FE257F1C5ACEFD00210C36 /* AES.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = AES.h; sourceTree = "<group>"; };
@@ -1043,6 +1046,8 @@
 		E6ADB9A411957099004E2BE5 /* wxWidgets */ = {
 			isa = PBXGroup;
 			children = (
+				6FF5D49D1DA14AB20032F5B6 /* passwordsubset.cpp */,
+				6FF5D49E1DA14AB20032F5B6 /* passwordsubset.h */,
 				E6F8DC1E1D132618007DFBEC /* mainView.cpp */,
 				E6E895981BD8C5E1009F3A8E /* TimedTaskChain.cpp */,
 				E6E895991BD8C5E1009F3A8E /* TimedTaskChain.h */,
@@ -1621,6 +1626,7 @@
 				E6B5C6451232B3D000AC8B45 /* dragbar.cpp in Sources */,
 				E6B5C6461232B3D000AC8B45 /* PWSDragBar.cpp in Sources */,
 				E61D6FA312617EFC0049FA2A /* MergeDlg.cpp in Sources */,
+				6FF5D4A11DA14EE80032F5B6 /* passwordsubset.cpp in Sources */,
 				E60F25D812C4ACEB001E63C4 /* ExternalKeyboardButton.cpp in Sources */,
 				E6A7ECAE12CDEAA40021D898 /* DbSelectionPanel.cpp in Sources */,
 				E6A7ECAF12CDEAA40021D898 /* PwsSync.cpp in Sources */,

